### DISCUSSION TITLE
Adding windows padding

### DISCUSSION THREAD

**Discussion by jochemcb** [pthom/imgui_bundle#245] [Q&A] [answered]
Hi,
I would like to add windows padding using Python, but I cannot find the solution.
The closest statement I could find is this;
imgui.push_style_var(imgui.Style.window_padding,ImVec2(50,50))
But the statement is giving a type error.

Is there a solution for adding windows padding?
Regards,
Jochem

**Comment by pthom** [comment]
Well, we have:

```python
def push_style_var(idx: StyleVar, val: ImVec2) -> None:
   ...

# Where
StyleVar = int

```

So, it requires an int which you can get with:
```python
imgui.push_style_var(imgui.Style.window_padding.value ,ImVec2(50,50))
```

**Comment by jochemcb** [comment]
Thanks for your fast reply!
I tried:
imgui.push_style_var(imgui.Style.window_padding.value ,ImVec2(50,50))
but it gives an error on : imgui.Style.window_padding.value
AttributeError: 'property' object has no attribute 'value'

**Reply by bgribble** [reply, marked as answer]
I think you want to use the `StyleVar_` and not `Style`, at least that's how I do it in my Python app.

Also, the `ImVec` types don't have to be used directly, you can use a Python tuple or list with the right number of elements instead. 

From my code base:
```
    imgui.push_style_var(imgui.StyleVar_.window_padding, (8, 8))
```

**Reply by jochemcb** [reply]
Thanks @bgribble That worked!
